src/ResponseTagger/Delegator/RestXLocationBCTagger.php

```diff
@@ -0,0 +1,33 @@
+<?php
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace EzSystems\PlatformHttpCacheBundle\ResponseTagger\Delegator;
+
+use EzSystems\PlatformHttpCacheBundle\ResponseConfigurator\ResponseCacheConfigurator;
+use EzSystems\PlatformHttpCacheBundle\ResponseTagger\ResponseTagger;
+use Symfony\Component\HttpFoundation\Response;
+
+class RestXLocationBCTagger implements ResponseTagger
+{
+    /**
+     * @var \EzSystems\PlatformHttpCacheBundle\ResponseTagger\ResponseTagger
+     */
+    private $restLocationBCTagger;
+
+    public function __construct(ResponseTagger $restLocationBCTagger)
+    {
+        $this->restLocationBCTagger = $restLocationBCTagger;
+    }
+
+    public function tag(ResponseCacheConfigurator $configurator, Response $response, $value)
+    {
+        // Rest API sets the X-Location-Id header. This needs to be changed according to http cache in use
+        if ($value === null && $response->headers->has('X-Location-Id')) {
+            $this->restLocationBCTagger->tag($configurator, $response, ['X-Location-Id' => $response->headers->get('X-Location-Id')]);
```

Comment:
Kind of duplicates #21, 

It seems a bit strange to use several layers of response taggers to map X-Location-id to xkey and remove it.

Reply:
and now it is also useless because of #25 I think